spec/actors/hyrax/actors/create_with_remote_files_actor_spec.rb

```diff
@@ -134,4 +134,114 @@
       expect(actor.send(:validate_remote_url, URI('https://example.com/test.txt'))).to be true
     end
   end
+
+  context 'when work is a valkyrie resource' do
+    let(:work) { valkyrie_create(:monograph) }
```

Comment:
These are the same tests as above, but run with a Valkyrie resource instead of AF Object.